src/tools/sssctl/sssctl_logs.c

```diff
@@ -309,43 +309,18 @@ errno_t sssctl_debug_level(struct sss_cmdline *cmdline,
     struct debuglevel_tool_ctx *ctx = NULL;
     struct poptOption long_options[] = {
         POPT_AUTOHELP
```

Comment:
As you're going to use  sss_tool_popt_ex(), please, remove the POPT_AUTOHELP from here, otherwise the help options are going to be printed twice.